builder/vmware/common/driver_workstation_unix.go

```diff
@@ -125,7 +125,7 @@ func workstationToolsIsoPath(flavor string) string {
 
 func workstationVerifyVersion(version string) error {
 	if runtime.GOOS != "linux" {
-		return fmt.Errorf("The VMware WS version %s driver is only supported on Linux, and Windows, at the moment. Your OS: %s", version, runtime.GOOS)
+		return fmt.Errorf("driver is only supported on linux, not %s", runtime.GOOS)
```

Comment:
Same remark here, while the implementation is OS-specific, from a user's perspective using Windows instead of an incompatible UNIX would work too.

Reply:
Resolved in 67054cd.